apps/nextjs/src/components/AppComponents/Chat/drop-down-section/action-drop-down.tsx

```diff
@@ -0,0 +1,114 @@
+import { Dispatch, RefObject, SetStateAction } from "react";
+
+import { aiLogger } from "@oakai/logger";
+import { OakP, OakRadioGroup } from "@oaknational/oak-components";
+import { $Enums, AilaUserModificationAction } from "@prisma/client";
+import { TextArea } from "@radix-ui/themes";
+
+import {
+  AdditionalMaterialOptions,
+  ModifyOptions,
+} from "./action-button.types";
+import { DropDownFormWrapper, FeedbackOption } from "./drop-down-form-wrapper";
+import { SmallRadioButton } from "./small-radio-button";
+
+const log = aiLogger("chat");
+
+type DropDownProps = {
+  sectionTitle: string;
+  options: ModifyOptions | AdditionalMaterialOptions;
+  selectedRadio: FeedbackOption<AilaUserModificationAction> | null;
+  setSelectedRadio: Dispatch<
+    SetStateAction<FeedbackOption<$Enums.AilaUserModificationAction> | null>
+  >;
+  isOpen: boolean;
+  setIsOpen: (open: boolean) => void;
+  setUserFeedbackText: (text: string) => void;
+  handleSubmit: (
+    option: FeedbackOption<AilaUserModificationAction>,
+  ) => Promise<void>;
+  buttonText: string;
+  userSuggestionTitle: string;
+  dropdownRef: RefObject<HTMLDivElement>;
+  id: string;
+};
+
+export const ActionDropDown = ({
+  sectionTitle,
+  options,
+  selectedRadio,
+  setSelectedRadio,
+  isOpen,
+  setIsOpen,
+  setUserFeedbackText,
+  handleSubmit,
+  buttonText,
+  userSuggestionTitle,
+  dropdownRef,
+  id,
+}: DropDownProps) => {
+  if (typeof options !== "object") return null;
```

Comment:
Same feedback again, do we expect the options to ever be in a different shape? If it shouldn't happen then it could be worth throwing